app/services/activitypub/fetch_featured_collection_service.rb

```diff
@@ -46,4 +46,8 @@ def process_items(items)
   def supported_context?
     super(@json)
   end
+
+  def local_follower
+    @local_follower ||= account.followers.local.without_suspended.first
```

Comment:
`did_you_mean` gem warned that there is no `account` method here. Should this be `@account`instead?

```
@local_follower ||= account.followers.local.without_suspended.first
                    ^^^^^^^
Did you mean?  @account
pid=69 tid=3l1w9 WARN: /app/app/services/activitypub/fetch_featured_collection_service.rb:51:in `local_follower'
/app/app/services/activitypub/fetch_featured_collection_service.rb:26:in `block in process_items'
/app/app/services/activitypub/fetch_featured_collection_service.rb:26:in `each'
```

Sorry that I don't know the code enough to come up with a failing test.

Reply:
Seems to have been fixed with #17326. Thanks!